### ISSUE TITLE
Problems with the quota in the channel on the server

### ISSUE THREAD

**Issue by artem59k** [BearWare/TeamTalk5#1360]
### Description

Hello. Is it possible to fix a bug that allows administrators to upload a file to a channel larger than the quota allowed. With this error, the administrator downloads files more than the quota set

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [X] iTeamTalk
- [X] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [X] Windows
- [X] macOS
- [X] Android
- [X] iOS
- [X] Linux

### Steps to reproduce problem

When an administrator uploads a file to a channel with an overflowing quota: the content is uploaded, and the administrator is not prohibited from further filling the server with files.

**Comment by bear101** [comment]
@CoBC Should we also limit the administrators to obey disk quota?

**Comment by CoBC** [comment]
Yes, I think. A quota can be specify for technical reasons (litle disk space available) so for me administrator shouldn't exceed this.

**Comment by artem59k** [comment]
Hello. I propose to improve the quota in the channel. Is there such a possibility to make a checkbox when creating a channel that will allow uploading files only by the administrator? In this case, the entire quota on the server can be used. Thanks.

**Comment by artem59k** [comment]
This will allow you to create really rooms for administrators in which there are no restrictions

**Comment by artem59k** [comment]
Hello. This idea is very bad. Many administrators did not approve of my proposal. Let the quota remain as it was before.
